packages/client/bin/cli.ts

```diff
@@ -727,9 +729,16 @@ async function run() {
   }
 
   const datadir = args.dataDir ?? Config.DATADIR_DEFAULT
-  const configDirectory = `${datadir}/${common.chainName()}/config`
+  const networkDir = `${datadir}/${common.chainName()}`
+  const configDirectory = `${networkDir}/config`
   ensureDirSync(configDirectory)
   const key = await Config.getClientKey(datadir, common)
+
+  // logFile is either filename or string "true"
+  if (typeof args.logFile === 'boolean') {
```

Comment:
How does this `typeof` call know it's a `boolean` as opposed to a string?

Reply:
the comment is misleading because of some refactoring i did:

`// logFile is either filename or boolean true/false`